vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -878,6 +878,15 @@ - (void)downloadUserModelForId:(NSString *)id completion:(void (^)(BOOL success,
   }];
 }
 
+- (void)getBrowsingHistory:(const int)max_count
+                   forDays:(const int)days_ago
+                  callback:(ads::GetBrowsingHistoryCallback)callback;
+{
+  // Not implemented for iOS
+  NSArray* history = @[];
+  callback(history);
```

Comment:
Since we're not bridging into iOS types at all you can just call the callback with the regular `std::vector` 🙂

I.e. `callback({})` or `callback(std::vector<std::string>())`

Reply:
fixed